src/app/domains/domain-detail/domain-detail.component.spec.ts

```diff
@@ -88,16 +88,16 @@ describe('DomainDetailComponent', () => {
     expect(routerSpy).toHaveBeenCalled()
   })
 
-  it('should read the domain certificate file contents on file upload', () => {
-    const obj = {
-      data: 'application/x-pkcs12;base64;domaincertdata'
-    }
-    const event = {
-      target: {
-        files: [new Blob([JSON.stringify(obj, null, 2)], { type: 'application/json' })]
-      }
-    }
+  // it('should read the domain certificate file contents on file upload', () => {
```

Comment:
Why are we leaving commented code in?

Reply:
it's a needed test, it just has no expectation --- @Mahendra-intel  will fix it in the domains test coverage PR